pyaedt/generic/DataHandlers.py

```diff
@@ -482,6 +489,39 @@ def to_aedt(code):
     return return_code
 
 
+def str_to_bool(s):
+    """Convert a ``"True"`` or ``"False"`` string to its corresponding Boolean value.
```

Comment:
```suggestion
    """Convert a string representing a true or false condition to its corresponding Boolean value.
```
I think that because the value to convert can be in various forms, we shouldn't tag as if it is ``"True"`` or ``"False"`` in descriptions.

Reply:
done